iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ReplaceTeePointByVoltageLevelOnLine.java

```diff
@@ -206,7 +211,7 @@ public void apply(Network network, boolean throwException,
             newLine2Adder.setBus1(bus2.getId());
             newLine2Adder.setConnectableBus1(bus2.getId());
         } else if (topologyKind == TopologyKind.NODE_BREAKER) {
-            BusbarSection bbs = getBusBarSectionFromVoltageLevel(tappedVoltageLevel, bbsOrBusId);
+            BusbarSection bbs = getBusBarSectionFromVoltageLevel(network, tappedVoltageLevel, bbsOrBusId);
```

Comment:
Can you use `tappedVoltageLevel.getNodeBreakerView().getBusbarSection(bbsOrBusId)` instead?

Reply:
it does not seem to work as expected as stated in the comment section

"tappedVoltageLevel.getNodeBreakerView().getBusbarSection(bbsOrBusId)" will return the busBarSection from the network even if it does not belong to "tappedVoltageLevel"

Reply:
Hm, interesting, thank you for your comment, it is a bug :) Florian's proposition is okay for me then, I will do a PR to fix the bug